cmd/podman/commit.go

```diff
@@ -33,7 +33,9 @@ var (
 			commitCommand.GlobalFlags = MainGlobalOpts
 			return commitCmd(&commitCommand)
 		},
-		Example: "CONTAINER [REPOSITORY[:TAG]]",
+		Example: `  podman commit -q --message "committing container to image" reverent_golick image-commited
```

Comment:
@TomSweeneyRedHat Adding spaces to the beginning of the Example:'s seems like a code smell that will bite us when using some of auto-generation tools.  E.g., https://github.com/spf13/cobra/blob/1e58aa3361fd650121dceeedc399e7189c05674a/doc/rest_docs.go#L90 is already adding spaces so your spaces could disturb what it is trying to do.

Reply:
@jwhonce here's what I know, and what I know is just a little!

With the code as it is in the code review, the example is shown on the command line as:

```
Usage:
  podman commit [flags]

Examples:
  podman commit -q --message "committing container to image" reverent_golick image-commited
  podman commit -q --author "firstName lastName" reverent_golick image-commited
  podman commit -q --pause=false reverent_golick image-commited
```
If I remove the two spaces as you suggest, I get:
```
Usage:
  podman commit [flags]

Examples:
podman commit -q --message "committing container to image" reverent_golick image-commited
  podman commit -q --author "firstName lastName" reverent_golick image-commited
  podman commit -q --pause=false reverent_golick image-commited
```
(just in case GitHub's md lives to beat me, the first example has no spacing unlike any other start of line under a category, the next lines do).

From what I understand of these changes, that output is controlled by the UsageTemplate() function in libpod/cmd/common.go (https://github.com/containers/libpod/blob/master/cmd/podman/common.go#L516)
which has:

```
Examples:
{{.Example}}{{end}}{{if .HasAvailableSubCommands}}
```

If I were to move `{{.Example}}{{end}}...` over by two spaces, but left the two spaces in the following example lines in commit.go, would that alleviate your concerns?  Or are you concerned with all of the preceeding spaces?